packages/vx-demo/src/components/ApiTable.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from 'react';
+import Markdown from 'react-markdown/with-html';
+
+type PropInfo = {
+  defaultValue?: { value?: unknown };
+  description?: string;
+  name: string;
+  required: boolean;
+  type?: { name: string };
+};
+
+type DocGenInfo = {
+  description?: string;
+  displayName?: string;
+  props: { [propName: string]: PropInfo };
+};
+
+type Props = {
+  anchorId?: string;
+  docgenInfo: DocGenInfo;
+};
+
+const alphaSort = (a: PropInfo, b: PropInfo) => a.name.localeCompare(b.name);
+
+export default function ApiTable({ anchorId, docgenInfo }: Props) {
+  const componentName = docgenInfo.displayName;
+
+  // required first, then abc order
+  const props = useMemo(() => {
+    const requiredProps = [];
+    const optionalProps = [];
+
+    Object.values(docgenInfo.props).forEach(prop => {
+      if (prop.required) {
+        requiredProps.push(prop);
+      } else {
+        optionalProps.push(prop);
+      }
+    });
+
+    return [...requiredProps.sort(alphaSort), ...optionalProps.sort(alphaSort)];
+  }, [docgenInfo]);
+
+  return (
+    <div className="api">
+      <h3>
+        <a id={anchorId} href={`#${anchorId}`}>
+          #
+        </a>
+        &lt;{componentName} /&gt;
+      </h3>
+      {props.map(prop => {
+        const id = `${componentName}_${prop.name}`;
+        return (
+          <div key={prop.name} className="prop">
+            <div className="title">
+              <span className="name">
+                <a id={id} href={`#${id}`}>
+                  #
+                </a>{' '}
+                <em>{componentName}</em>.<strong>{prop.name}</strong>
+              </span>
+              {prop.type && <code>{prop.type.name}</code>}
+              <span className="required">{prop.required ? 'required' : 'optional'}</span>{' '}
+            </div>
+            <div className="description">
+              <Markdown
+                source={`${prop.description}${
+                  prop.defaultValue
+                    ? `\n\nDefault \`${String(prop.defaultValue.value) || '""'}\``
+                    : ''
+                }`}
+              />
+            </div>
+          </div>
+        );
+      })}
+      <style jsx>{`
+        .prop {
+          border-top: 1px solid #eaeaea;
+          padding: 1em 0.5em;
+          line-height: 1.2em;
+          vertical-align: middle;
+        }
+        .title > :not(:last-child) {
+          margin-right: 12px;
+        }
+        .description > :global(p) {
```

Comment:
`:global` is needed because `styled-jsx` can't figure out that `Markdown` creates a `p` so the style doesn't apply